proto/multipoolermanagerservice.proto

```diff
@@ -0,0 +1,113 @@
+    returns (multipoolermanagerdata.SetStandbyPrimaryConnInfoResponse);
+
+  // StartStandbyReplication starts WAL replay on standby (calls pg_wal_replay_resume)
+  rpc StartStandbyReplication(multipoolermanagerdata.StartReplicationRequest)
+    returns (multipoolermanagerdata.StartReplicationResponse);
+
+  // StopStandbyReplication stops WAL replay on standby (calls pg_wal_replay_pause)
+  rpc StopStandbyReplication(multipoolermanagerdata.StopStandbyReplicationRequest)
+    returns (multipoolermanagerdata.StopStandbyReplicationResponse);
+
+  // StandbyReplicationStatus gets the current replication status of the standby
+  rpc StandbyReplicationStatus(multipoolermanagerdata.StandbyReplicationStatusRequest)
+    returns (multipoolermanagerdata.StandbyReplicationStatusResponse);
+
+  // ResetStandbyReplication resets the standby's connection to its primary by clearing primary_conninfo,
+  // reloading PostgreSQL configuration, and terminating the WAL receiver process. This effectively
+  // disconnects the replica from the primary and prevents it from acknowledging commits, making it
+  // unavailable for synchronous replication until reconfigured.
+  rpc ResetStandbyReplication(multipoolermanagerdata.ResetStandbyReplicationRequest)
+    returns (multipoolermanagerdata.ResetStandbyReplicationResponse);
+
+  // ConfigureSynchronousReplication configures PostgreSQL synchronous replication settings
+  // including synchronous_commit, synchronous_standby_names, and synchronization method (ANY/FIRST/quorum)
+  rpc ConfigureSynchronousReplication(multipoolermanagerdata.ConfigureSynchronousReplicationRequest)
+    returns (multipoolermanagerdata.ConfigureSynchronousReplicationResponse);
+
+  // PrimaryStatus gets the status of the leader server
+  rpc PrimaryStatus(multipoolermanagerdata.PrimaryStatusRequest)
+    returns (multipoolermanagerdata.PrimaryStatusResponse);
+
+  // PrimaryPosition gets the current LSN position of the leader
+  rpc PrimaryPosition(multipoolermanagerdata.PrimaryPositionRequest)
+    returns (multipoolermanagerdata.PrimaryPositionResponse);
+
+  // StopReplicationAndGetStatus stops PostgreSQL replication and returns the status
+  rpc StopReplicationAndGetStatus(multipoolermanagerdata.StopReplicationAndGetStatusRequest)
+    returns (multipoolermanagerdata.StopReplicationAndGetStatusResponse);
+
+
+  // Multigres HA 
+  // The following methods are used by MultiOrch to 
+  // perform HA operations (reparents and failovers)
+
+  // ChangeType changes the pooler type (LEADER/FOLLOWER)
+  rpc ChangeType(multipoolermanagerdata.ChangeTypeRequest)
+    returns (multipoolermanagerdata.ChangeTypeResponse);
+
+  // GetFollowers gets the list of follower servers
+  rpc GetFollowers(multipoolermanagerdata.GetFollowersRequest)
+    returns (multipoolermanagerdata.GetFollowersResponse);
+
+  // DemoteLeader demotes the current leader server
+  rpc DemoteLeader(multipoolermanagerdata.DemoteLeaderRequest)
```

Comment:
Drop Leader and Follower from RPCs. The rationale is that you are invoking an RPC on a pooler which is already a leader/follower. There's no need to have them in the function names.